src/components/ArticleCard.jsx

```diff
@@ -11,6 +12,7 @@ const ArticleCard = ({ article }) => {
             <button><Link to={`/articles/${article.article_id}`}>
                 View
             </Link></button>
+            <Comments />
```

Comment:
Rendering Comments here is firing 36 requests for the comments without the review_id. You don't need to render the comments on the article card.